src/locale/i18n.ts

```diff
@@ -166,6 +167,14 @@ export async function dynamicActivate(locale: AppLanguage) {
       ])
       break
     }
+    case AppLanguage.km: {
+      i18n.loadAndActivate({locale, messages: messagesKm})
+      await Promise.all([
+        import('@formatjs/intl-pluralrules/locale-data/km'),
```

Comment:
```suggestion
        import('@formatjs/intl-datetimeformat/locale-data/km'),
        import('@formatjs/intl-pluralrules/locale-data/km'),
```

Following #6742 being merged, could you add this line please?

This will ensure that the date and time for posts are localized correctly on all platforms. Thanks!